### ISSUE TITLE
CPU Usage

### ISSUE THREAD

**Issue by Icehunter** [felixhageloh/uebersicht#210]
I've made a very simple widget that has a animated gif setup in a div.

Full screenshot here:
![image](https://cloud.githubusercontent.com/assets/677341/16064244/37700dd2-3254-11e6-9534-23750c098035.png)

Widget:
```coffee
# This is a simple example Widget, written in CoffeeScript, to get you started
# with Übersicht. For the full documentation please visit:
#
# https://github.com/felixhageloh/uebersicht
#
# You can modify this widget as you see fit, or simply delete this file to
# remove it.

# this is the shell command that gets executed every time this widget refreshes
command: "whoami"

# the refresh frequency in milliseconds
refreshFrequency: 86400000

# render gets called after the shell command has executed. The command's output
# is passed in as a string. Whatever it returns will get rendered as HTML.
render: (output) -> """
  <div class="coding-cat">
    <img src="coding-cat/coding-cat.gif" />
    <div class="tag-line">
      such is life...
    </div>
  </div>
"""

# the CSS style for this widget, written using Stylus
# (http://learnboost.github.io/stylus/)
style: """
  .coding-cat {
    position: absolute;
    top: 100px;
    left: 100px;
    height: 25vh;
    transform: rotate(-15deg);
  }
  .coding-cat .tag-line {
    transform: rotate(15deg);
    color: #FFF;
    padding-left: 8vw;
  }
  img
    width: inherit;
    height: inherit;
    border: 5px solid #FFF;
    border-radius: 10%;
    box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.75);
"""
```

I've attached image used as well.  Am I doing something wrong in that it uses 50+% cpu?
![coding-cat](https://cloud.githubusercontent.com/assets/677341/16064266/70e689f6-3254-11e6-8f03-fc6aeb59c5f3.gif)

**Comment by felixhageloh** [comment]
no as far as I can tell you are not doing anything wrong; I was able to reproduce the issue. I can only guess that it is an issue with WebView - it is often a lot slower than Safari. I'm hoping to update to the latest WKWebview soon and hopefully this will fix this issue.

As final small tip: if you don't want to run a command for your widget you can simply set `refreshFrequency: false` and omit the command altogether.

**Comment by felixhageloh** [comment]
This should be fixed in version 1.0.42 released today
